docs/docsite/_static/core.css

```diff
@@ -22,9 +22,9 @@
  * / is given the class .value-separator
  * required is given the class .value-required
  * added in x.y is given the class .value-added-in
- *//*! The extra .rst-content is so this will override rtd theme */.rst-content table.documentation-table td{vertical-align:top}table.documentation-table td:first-child{white-space:nowrap;vertical-align:top}table.documentation-table td:first-child p:first-child{font-weight:700;display:inline}/*! This is now redundant with above position-based styling *//*!
+ *//*! The extra .rst-content is so this will override rtd theme */.rst-content table.documentation-table td{vertical-align:top}table.documentation-table td:first-child p:first-child{font-weight:700;display:inline}/*! This is now redundant with above position-based styling *//*!
```

Comment:
Instead of changing this file, it looks like it should be just removed.

Reply:
Ref: https://github.com/ansible/ansible/pull/75056

Reply:
Gone after the rebase.